matrix_client/client.py

```diff
@@ -177,12 +278,25 @@ def _sync(self, limit=1):
             pass
 
     def get_user(self, user_id):
+        """ Return a User by their id.
+            NOTE: This function only returns a user object, it does not verify
+            the user with the Home Server.
```

Comment:
Is the indentation here A Thing? Either way, there should be a blank line between summary and note